src/middleware/mod.rs

```diff
@@ -55,7 +54,7 @@ pub fn build_middleware(app: Arc<App>, endpoints: R404) -> MiddlewareBuilder {
     }
 
     if env != Env::Test {
-        m.add(LogRequests(log::LogLevel::Info));
+        m.around(log_request::LogRequests::default());
```

Comment:
With around middleware, you may want to move this to the bottom of the file to ensure that it is the outermost around wrapper.  Otherwise, this won't log blacklisted ips or the static index.html served on non-api endpoints.

Alternatively, I think this can be converted to before/after Middleware.  If I understand correctly, the around middleware should only be necessary when we want to rewrite the request or to short-circuit other AroundMiddleware or the innermost handler by returning a response early.  A before handler can short-circuit by returning an `Err(_)`, but cannot return early with a response.  Here we only need to capture the start time on the way down the middleware stack.